### PR TITLE
AST-1657 - Modern my-acount compatibility from theme side

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- As old CSS coming from theme which we don't want in modern look, excluding that much
- likewise compatibility adding from theme side in this PR

### Types of changes
- New feature (non-breaking change )

### How has this been tested?
- 

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: sass/site/compatibility/woocommerce/woocommerce.scss [hunk 1]
```diff
@@ -3402,8 +3392,7 @@ button.pswp__button--zoom:hover {
 /**
  * My Account Style
  */
-.woocommerce-MyAccount-navigation-link {
-
+body:not(.ast-modern-woo-account-page) .woocommerce-MyAccount-navigation-link {
```
Comment (premanshup) [inline sass/site/compatibility/woocommerce/woocommerce.scss:3395]: @imnavanath Same comment as below.

File: assets/css/unminified/compatibility/woocommerce/woocommerce-grid.css [hunk 2]
```diff
@@ -4075,25 +4065,35 @@ button.pswp__button--zoom:hover {
...
 
+body:not(.ast-modern-woo-account-page) .woocommerce form.login,
+body:not(.ast-modern-woo-account-page) .woocommerce form.checkout_coupon,
+body:not(.ast-modern-woo-account-page) .woocommerce form.register {
+  border: 1px solid #d3ced2;
+  padding: 20px;
+  margin: 2em 0;
+  text-align: left;
+  border-radius: 5px;
+}
```
Comment (premanshup) [inline assets/css/unminified/compatibility/woocommerce/woocommerce-grid.css:4095]: @imnavanath Same comment as below.

Comment (imnavanath) [reply assets/css/unminified/compatibility/woocommerce/woocommerce-grid.css:4095]: Done in the PR - https://github.com/brainstormforce/astra/pull/4217

File: sass/site/compatibility/woocommerce/woocommerce.scss (hunk 1 above)
Comment (imnavanath) [reply sass/site/compatibility/woocommerce/woocommerce.scss:3395]: Done in the PR - https://github.com/brainstormforce/astra/pull/4217

File: sass/site/compatibility/woocommerce/woocommerce.scss [hunk 3]
```diff
@@ -3419,8 +3408,18 @@ button.pswp__button--zoom:hover {
...
+	form.login,
+	form.checkout_coupon,
+	form.register {
+		border: 1px solid darken( $secondary, 10% );
+		padding: 20px;
+		margin: 2em 0;
+		text-align: left;
+		border-radius: 5px;
+	}
 }
```
[the remark this answers is not in the document]
Comment (imnavanath) [reply sass/site/compatibility/woocommerce/woocommerce.scss:3423]: Done in the PR - https://github.com/brainstormforce/astra/pull/4217

**Pull request merged**